### PR TITLE
Fixes #31197 - ansible galaxy URL updates

### PR DESCRIPTION
**Pull request by jjeffers**

### CODE REVIEW

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/new/new-repository.controller.js [hunk 1]
```diff
@@ -141,10 +141,8 @@ angular.module('Bastion.repositories').controller('NewRepositoryController',
 
             $scope.collectionURLPopover = $sce.trustAsHtml("You can sync collections utilizing just the url:<br/>" +
           "<b>1. For all collections in Ansible Galaxy:</b><br/>" +
-          "https://galaxy.ansible.com/api/v2/collections <br/>" +
-          "<b>2. For specific collection with URL filtering:</b><br/>" +
-          "https://galaxy.ansible.com/api/v2/collections/testing/k8s_demo_collection <br/>" +
-          "<b>3. For specific collections with Requirements.yml:</b><br/>" +
+          "https://galaxy.ansible.com/api/<br/>" +
```
Comment (sjha4) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/new/new-repository.controller.js:144]: Think https://galaxy.ansible.com/ is used for mirroring although the last I tried to mirror, it failed on some collection in galaxy..Not sure we want to advertise mirroring?

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/new/new-repository.controller.js [hunk 2]
```diff
@@ -141,10 +141,8 @@ angular.module('Bastion.repositories').controller('NewRepositoryController',
...
+          "<b>2. For specific collections with Requirements.yml:</b><br/>" +
           "Use base URL https://galaxy.ansible.com/ and specify requirements.yml below to specify collections");
 
             $scope.requirementPopover = $sce.trustAsHtml("To learn more about requirement.yml specification, visit <a href='https://docs.ansible.com/ansible/devel/dev_guide/collections_tech_preview.html#install-multiple-collections-with-a-requirements-file' target=\"_blank\">documentation </a>");
```
Comment (sjha4) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/new/new-repository.controller.js:148]:
```suggestion
            $scope.requirementPopover = $sce.trustAsHtml("To learn more about requirement.yml specification, visit <a href='https://docs.ansible.com/ansible/latest/galaxy/user_guide.html#install-multiple-collections-with-a-requirements-file' target=\"_blank\">documentation </a>");
```

Comment (jlsherrill) [reply engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/new/new-repository.controller.js:148]: yeah i just noticed the old url throws a 404, +1 to the update

**Comment by jlsherrill** [comment]
Tested capsule sync with no issues!  so just some small text updates needed

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/new/new-repository.controller.js (hunk 1 above)
Comment (jjeffers) [reply engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/new/new-repository.controller.js:147]: @sjha4 you mean mirroring all of Galaxy? Do we know why this was presented as a tool tip in the first place?

Comment (sjha4) [reply engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/new/new-repository.controller.js:147]: Ya..That was supposed to be a use case for collections support..Mirroring all of galaxy. I don't mind leaving that as advertised. Only issue was last I tried it, it failed on some collection in the galaxy. Think we'll have to test it better so it doesn't stay broken.

Comment (jjeffers) [reply engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/new/new-repository.controller.js:147]: @sjha4 @jlsherrill aside from the comments re the galaxy url being listed... any other items we need to adjust? I think @jeremylenz is waiting on this for 4.1 picks.

Comment (jlsherrill) [reply engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/new/new-repository.controller.js:147]: maybe we should discuss it more with the pulp-ansible team and merge this as is.  ACK from me

Comment (sjha4) [reply engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/new/new-repository.controller.js:147]: Ack +1

**Review by sjha4 — APPROVED** [review]

**Pull request merged**